CONTRIBUTING.md

```diff
@@ -49,17 +49,18 @@ working on your contributions.
 
 ### Format of the Commit Message
 
-We follow a rough convention for commit messages that is designed to answer two
-questions: what changed and why. The subject line should feature the what and
-the body of the commit should describe the why.
+The commit messages should be designed to answer two questions: what changed and why. 
+
+The subject line should consist of the modified subsytem and an imperative sentence
```

Comment:
subytem -> subsystem
although it strikes me that we should attempt to define what this is